### DISCUSSION TITLE
How is volumetric soil content calculated?

### DISCUSSION THREAD

**Discussion by diegodinamarca** [mhm-ufz/mHM#63] [Q&A] [answered]
Hello mHM team, I've been working with the volumetric soil moisture output (outputFlxState 4), trying to compare it with soil moisture observations, so checking the output reference this volumetric content is calculated relative to a saturated soil moisture value (L1_soilMoist / L1_soilMoistSat). So my question is how is this saturated soil moisture value calculated?

**Comment by rohinikumar82** [comment, marked as answer]
L1_soilMoistSat - is a model parameter that is estimated via MPR - regionalization via a set of pedo-transfer functions and then aggregated via a defined upscaling technique. More on this, can be read in the main mHM paper reference (Samaniego et al., 2010) or here in Livneh et al., 2015 (Livneh, B., Kumar, R. and Samaniego, L., 2015. Influence of soil textural properties on hydrologic fluxes in the Mississippi river basin. Hydrological Processes, 29(21), pp.4638-4655.)
Hope this helps.
-Rohini

**Reply by diegodinamarca** [reply]
ty! i'll check those papers

**Reply by pallavhydro** [reply]
Hi @diegodinamarca 

If the discussion here solved your issue, I'd appreciate you mark the response that solved the issue.

Thanks in advance.
Pallav
